index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portfolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="page-preview">
+      <nav>
+        <div class="left-logo">Gordon Otieno</div>
+        <div class="humburger"><img src="./icons/hamburger.png" alt="logo"></div>
+      </nav>
+      <div class="intro">
+        <h1>Hay There I'm Gordon</h1>
+        <h2>I am Software Developer</h2>
+        <p>I can help you develop develop application, Build websites, provide you with hosting servises and many more.
+           Please feel free to contact me on my social media accounts below</p>
+        </div>
+         <div class="social-media">
+          <div class="linkedin-icon"><img src="/icons/github.png" alt="linkedin"></div>
+          <div class="angelist-icon"><img src="/icons/angelist.png" alt="angelist"></div>
```

Comment:
- [REQUIRED] Please, try to replace the `div` tag with a `ul` and putt all icons inside as child `li` elements. This is more semantic.

```html
<ul class="social-media">
  <li>
    <a href="#"> <img src="images/icons/github.png" alt=""> </a>
  </li>
  <li>
    <a href="#"> <img src="images/icons/linkedin.png" alt=""> </a>
  </li>
 <!-- rest of code -->
</ul>
```